src/api/vdom.js

```diff
@@ -295,6 +295,21 @@ function newElementVoid (tag, ...args) {
 // Text override ensures their calls can queue if using function helpers.
 const newText = wrapIdomFunc(text);
 
+export function child (ch) {
+  const ctype = typeof ch;
+  if (ctype === 'function') {
+    ch();
+  } else if (ctype === 'string' || ctype === 'number') {
+    newText(ch);
+  } else if (Array.isArray(ch)) {
+    ch.forEach(child);
+  } else if (ctype === 'object' && ch.$$typeof) {
+    const { key, props, ref, type } = ch;
```

Comment:
I completely understand that you need to support React because you are using it at work, but IMHO this shouldn't be in core `skate`. this interop should be provided by some plugin system via separate package in the future. Relying on some 3rd party implementation is compatibility and maintenance hell in my experience.